E/Entwine/bundled/patches/mingw-remove-msvc-shlwapi-lib-search.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/cmake/system.cmake b/cmake/system.cmake
+index 15f8021..de51fae 100644
+--- a/cmake/system.cmake
++++ b/cmake/system.cmake
+@@ -1,6 +1,5 @@
+ if (WIN32)
+     include(${CMAKE_DIR}/windows_compiler_options.cmake)
+-    find_library(SHLWAPI Shlwapi.lib)
```

Comment:
Instead of removing this line, would it work with something like
```cmake
find_library(SHLWAPI shlwapi)
```
?  The extension is controlled by [`CMAKE_FIND_LIBRARY_SUFFIXES`](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_LIBRARY_SUFFIXES.html), the prefix (e.g. `lib`) by [`CMAKE_FIND_LIBRARY_PREFIXES`](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_LIBRARY_PREFIXES.html).  If this is set correctly, you probably shouldn't need the patch `mingw-shwlapi-hardcode.patch`

Reply:
Yes, that works for me locally. I also like that I have to remember how to spell `shlwapi` correctly in fewer files :laughing: